meta-swupd/recipes-core/swupd-client/swupd-client/Fix-build-failure-on-Yocto.patch

```diff
@@ -6,7 +6,7 @@ Subject: [PATCH] Fix build failure on Yocto
 On install phase certificate files are being installed twice as included in
 _DATA twice. We can use EXTRA_DIST than dist_.
 
-Upstream-Status: Inappropriate
+Upstream-Status: Accepted
```

Comment:
I think this counts as a backport also? See: http://www.openembedded.org/wiki/Commit_Patch_Message_Guidelines#Patch_Header_Recommendations